Framework/Beamline/inc/MantidBeamline/EigenTest.h

```diff
@@ -0,0 +1,13 @@
+#ifndef MANTID_EIGENTEST_H
```

Comment:
Can we remove the header and simply put everything in the test? That would avoid polluting the library and no one will mind keeping the test around.